avogadro/quantumio/gaussiancube.cpp

```diff
@@ -0,0 +1,123 @@
+/******************************************************************************
+
+  This source file is part of the Avogadro project.
+
+  Copyright 2015 Barry E. Moore II
+  Copyright 2013 Kitware, Inc.
```

Comment:
Sorry to be a pain, you don't need to add Kitware if the entire file was written by you. As a side note I have been reading up a little on this, and we may want to consider removing these lines - all of this information is available (and properly tracked) in the git history. I am a fan of minimal headers, source releases would lack it, but can still refer to the git repository (with tagged releases).